src/scenes/home/personalLinks.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { personalLinks } from "../../data/personalLinks";
+import { PersonalLinksItem } from "./personalLinksItem";
+import styles from "./personalLinks.module.scss";
+
+export function PersonalLinks() {
+  const createList = () => {
```

Comment:
is the separate function necessary? i would put it inline, inside div tag

Reply:
Done!